File: Rok2/Java/Obliczenia/Stala.java
package obliczenia;

/**
 * @author karol
 *Klasa reprezentująca stałą
 */
public class Stala extends Wyrazenie {
    public final String znak;
    public double wartosc;

    public Stala(String znak , double wartosc)
    {
        this.znak=znak;
        this.wartosc=wartosc;
    }

    @Override
    public double Oblicz() {
        return wartosc;
    }

    @Override
    public String toString() {
        return znak;
    }
}

File: Rok1/C++/6/Wyrazenia/Stala.cpp
#include "Stala.h"
#include "Wyrazenie.h"
#include <string>
using namespace std;

double stala::oblicz()const {
    return value;
}

string stala::opis() const  {

    string s =to_string(value);
    return s;
}

stala::stala(int value) : value(value) { }


int stala::getValue() {
    return value;
}

std::string stala::formatuj_wyrazenie() const {
    return std::to_string(value);
}
